.travis_quickstart.sh

```diff
@@ -36,90 +39,98 @@ cd ${DIST_BIN_DIR}
 # Test quick-start-batch
 bin/quick-start-batch.sh &
 PID=$!
-sleep 60
-COUNT_STAR_RES=`curl -X POST --header 'Accept: application/json'  -d '{"sql":"select count(*) from baseballStats limit 1","trace":false}' http://localhost:8000/query/sql | jq '.resultTable.rows[0][0]'`
-if [ "${COUNT_STAR_RES}" -ne 97889 ]; then
-  echo 'Batch Quickstart: Incorrect result for count star query.'
+
+PASS=0
+sleep 30
```

Comment:
Can we avoid this sleep if possible and retry before the cluster start responding? Same for real-time and hybrid

Reply:
I've test it out, it requires more than 30 secs to bring up quickstart ready for query. if we removed this line, we will see more garbage logs of curl failures

Reply:
and hybrid quickstart requires about 1 minute to start :(